changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+      {
+        "Id": "12b6de5f-fe2d-4593-9f24-d95543e71199",
+        "ApiChange": "Resource",
+        "ChangedApiName": "application",
+        "ChangeType": "Change",
+        "Description": "Added limits on the *requiredResourceAccess* property for the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta) resource. No more than 50 required APIs, and no more than 400 total required permissions may be included.",
+        "Target": "requiredResourceAccess"
+      }
+     ],
+     "Id": "12b6de5f-fe2d-4593-9f24-d95543e71199",
+     "Cloud": "prd",
+     "Version": "beta",
+     "CreatedDateTime": "2021-09-27T19:51:16.2331125Z",
+     "WorkloadArea": "Identity and access",
+     "SubArea": "Applications"
+    },
+    {
+      "ChangeList": [
+      {
+        "Id": "1863357d-b68c-4f46-8a7e-dd4f8ebbd68b",
+        "ApiChange": "Resource",
+        "ChangedApiName": "application",
+        "ChangeType": "Change",
+        "Description": "Added limits on the *requiredResourceAccess* property for the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-v1.0) resource. No more than 50 required APIs, and no more than 400 total required permissions may be included.",
+        "Target": "requiredResourceAccess"
```

As per the Example 10 in the Changelog guidance
```suggestion
        "Id": "1863357d-b68c-4f46-8a7e-dd4f8ebbd68b",
        "ApiChange": "Resource",
        "ChangedApiName": "requiredResourceAccess",
        "ChangeType": "Change",
        "Description": "Added limits on the *requiredResourceAccess* property of the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-v1.0) resource. No more than 50 required APIs, and no more than 400 total required permissions may be included.",
        "Target": "application"
```